templates/repo/bare.tmpl

```diff
@@ -18,14 +18,14 @@
 										{{if UseHTTPS}}HTTPS{{else}}HTTP{{end}}
 									</button>
 								{{end}}
-								{{if not $.DisableSSH}}
+								{{if and (not $.DisableSSH) (or $.IsSigned $.ExposeAnonSSH)}}
```

Comment:
For bare repository check is not needed as there is already check to show clone url only if user is `IsRepositoryAdmin`